### PR TITLE
show no model tips when no model in preview

### PR DESCRIPTION
**Pull request by chichinohaha**
Re: cocos-creator/3d-tasks#
image

Changelog:
 *

### CODE REVIEW

File: editor/inspector/assets/fbx/preview.js [hunk 1]
```diff
@@ -441,15 +449,17 @@ exports.update = async function(assetList, metaList) {
         this.splitClipIndex = 0;
         const clipInfo = animation.methods.getCurClipInfo.call(this);
         await this.onEditClipInfoChanged(clipInfo);
+    } else {
+        // no animation
+        this.$.noModel.hidden = false;
+        this.$.container.hidden = true;
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/preview.js]: use `updatePanelHidden()` method

File: editor/inspector/assets/fbx/preview.js [hunk 2]
```diff
@@ -508,6 +518,12 @@ exports.close = function() {
 };
 
 exports.methods = {
+    updatePanelHidden() {
+        const previewDirty = !!this.isPreviewDataDirty;
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/preview.js]: panel element visible state nothing to do with 'this.isPreviewDataDirty'

File: editor/inspector/assets/fbx/preview.js [hunk 3]
```diff
@@ -281,7 +293,7 @@ exports.$ = {
 };
 
 const Elements = {
-    container: {
+    dragAread: {
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/preview.js]: unneccesary to modify this code

**Comment by chichinohaha** [comment]
重新提交一个pr

**Pull request closed without merging**